src/app/(sidebar)/explore-endpoints/[[...pages]]/page.tsx

```diff
@@ -127,22 +146,27 @@ export default function ExploreEndpoints() {
     // Clear form and errors if navigating to another endpoint page. We don't
     // want to keep previous form values.
     if (currentEndpoint && currentEndpoint !== currentPage) {
-      resetParams();
-      setFormError({});
+      resetStates();
     }
-  }, [currentPage, currentEndpoint, updateCurrentEndpoint, resetParams]);
+  }, [currentEndpoint, currentPage, resetStates, updateCurrentEndpoint]);
 
   useEffect(() => {
     // Save network for endpoints if we don't have it yet.
     if (network.id && !endpointNetwork.id) {
       updateNetwork(network as Network);
       // When network changes, clear saved params and errors.
     } else if (network.id && network.id !== endpointNetwork.id) {
-      resetParams();
-      setFormError({});
+      resetStates();
       updateNetwork(network as Network);
     }
-  }, [endpointNetwork.id, network, resetParams, updateNetwork]);
+  }, [endpointNetwork.id, network, resetStates, updateNetwork]);
+
+  // Scroll to response
+  useEffect(() => {
+    if (isSuccess || isError) {
+      responseEl?.current?.scrollIntoView({ behavior: "smooth" });
```

Comment:
this only works on the first submit call. it doesn't scroll down on subsequent calls

Reply:
No, it's a bug. I forgot to reset the response when the request URL changes. Thanks for catching it! :bow: